app/routers/api/storage.py

```
@@ -237,6 +256,18 @@ def update_storage_location(
     db.commit()
     db.refresh(location)
 
+    # Trigger background jobs after commit
+    if trigger_encryption_job:
+        logger.info(
+            f"Encryption enabled for location {location.name}. Queuing background encryption job."
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)